bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -1911,19 +1885,18 @@ bool ReplicaImp::tryToEnterView() {
 
   std::vector<PrePrepareMsg *> prePreparesForNewView;
 
-  LOG_INFO(GL,
-           "**************** Calling to viewsManager->tryToEnterView(curView="
-               << curView << ", lastStableSeqNum=" << lastStableSeqNum << ", lastExecutedSeqNum=" << lastExecutedSeqNum
-               << ")");
-
-  bool succ = viewsManager->tryToEnterView(curView, lastStableSeqNum, lastExecutedSeqNum, &prePreparesForNewView);
+  bool enteredView =
+      viewsManager->tryToEnterView(curView, lastStableSeqNum, lastExecutedSeqNum, &prePreparesForNewView);
 
-  if (succ)
+  LOG_INFO(GL,
+           "**************** Calling to viewsManager->tryToEnterView "
```

Comment:
Maybe this log statement should be before the actual call to tryToEnterView

Reply:
I wanted to include the result: `enteredView`. I'll change the message from `Calling` to `Called`.